### DISCUSSION TITLE
Kong performance with ARM instances

### DISCUSSION THREAD

**Discussion by shettyh** [Kong/kong#11074] [Help]
Hi,

Planning to migrate to Graviton 2 instances from Intel, but the performance results are not promising. Setup is not running directly in bare metal but on kubernetes. With similar speced Graviton 2 instance we are seeing about 50% throughput drop, are there any recommendation on any tuning we can do ? First of all is this expected ?

Using: Kong 2.8.3

Thanks

**Comment by baonq-me** [comment]
`Graviton 2` is a weaker chip than Intel or AMD, so a 50% throughput drop on benchmarks is expected with the exact core count. But we often don't use the gateway in extreme conditions like that, so `Graviton 2` will help reduce cost.

@shettyh, can you share your instance type and how you set up your benchmark?
